packages/@aws-cdk/aws-appconfig-alpha/README.md

```diff
@@ -330,8 +330,8 @@ declare const alarm: cloudwatch.Alarm;
 new appconfig.Environment(this, 'MyEnvironment', {
   application,
   monitors: [
-    {alarm},
-  ]
+    appconfig.Monitor.fromCloudWatchAlarm(alarm),
+  ],
 });
 ```
 
```

Comment:
We should have an example for `fromCfnMonitorsProperty` unless it is not the recommended approach. Even then the readme could have a line saying that it is an option but not the recommended approach

Reply:
Updated